test/matrixvariates.jl

```diff
@@ -322,6 +322,13 @@ function test_special(dist::Type{MatrixNormal})
             end
         end
     end
+    @testset "Non-allocating sampling" begin
+        # #2012: we can sample without allocations
+        M, U, V = _rand_params(MatrixNormal, Float64, 5, 5)
+        noallocD = MatrixNormal(M, cholesky!(Symmetric(U, :L)), cholesky!(Symmetric(V, :U)))
+        output = Matrix{Float64}(undef, size(noallocD))
+        @test iszero(((d, out) -> @allocated(rand!(d, out))(noallocD, output)))
```

Comment:
Sorry, it seems I messed up the order of the closing brackets in my previous suggestion. I think the following should be correct:
```suggestion
        @test iszero(((d, out) -> @allocated(rand!(d, out)))(noallocD, output))
```

Reply:
Indeed, I should've read more carefully.